src/embedded.js

```diff
@@ -539,28 +539,6 @@
 
             this.iframe.setAttribute('height', windowDims.heightRaw);
 
-            // Add or remove the 'cancel' button
-            if (!this.isInPage && (params['allowCancel'] === true || params['allowCancel'] === undefined) && !this.cancelButton) {
-                this.cancelButton = document.createElement('a');
-                this.cancelButton.setAttribute('id', 'hsEmbeddedCancel');
-                this.cancelButton.setAttribute('href', 'javascript:;');
-                this.cancelButton.onclick = function(){
-                    // Close iFrame
-                    HelloSign.close();
-                    // Send 'cancel' message
-                    if (messageListener) {
-                        l('Reporting cancelation');
-                        messageListener({
-                            'event': HelloSign.EVENT_CANCELED
-                        });
-                    }
-                };
-                this.wrapper.appendChild(this.cancelButton);
-            }
-            else if (!params['allowCancel'] && this.cancelButton) {
-                this.wrapper.removeChild(this.cancelButton);
-            }
```

Comment:
@chris-hellosign has a point. Might be worth giving this a test with UX v1 just to see if this indeed still works, for that case.

If it has issues, we can add a switch to detect the UX version and just stick the above code in that conditional block.